doc/api_review/core/keyAddName.md

```diff
@@ -103,10 +103,11 @@
 
 - [ ] Function code is fully covered by tests  
        - [ ] check if we can remove uncovered code  
```

Comment:
https://github.com/ElektraInitiative/libelektra/blob/master/src/libs/elektra/keyname.c#L593

is this code really unused? can we just remove it?

Reply:
I think this is used by the mmap cache. @mpranj can you confirm and do you have an idea why this code is untested? (Maybe because we don't do coverage tests on the mmap(storage) tests?)

Reply:
It is possible that the lines are simply not covered by the main `debian-buster-full` job. I'd be happy to add test cases to cover this code directly.

It is **definitely** a case which can happen. I systematically went through all cases where memory within a mmapped-region could be freed/reallocated/etc. Some corner cases were discovered by just running all tests with mmapstorage as default storage (we also have a CI job for that). Maybe the mentioned lines are even covered in this job.